clientV2/components/modals/Modal.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ModalWrapper = props => {
+  const { heading, children, openModal, closeModal } = props;
+  let className = 'modal';
+
+  if (!openModal) {
+    className += ' displayOff';
+  } else {
+    className += ' displayOn';
+  }
+  return (
+    <div className={className}>
+      <div className="modal_content">
+        <header className="modal_header">
+          <p>{heading}</p>
```

Comment:
I think there will be need for className on the p tag, with props passed into it. This will create room for additional or optional style that is not handle here.